ripple/src/main/webapp/posts.js

```diff
@@ -20,12 +20,11 @@ function saveCaption() {
   console.log("local storage caption: " + localStorage.getItem("caption"));
 }
 
-function addPostButtonDisplay() {
+/* If the user type is a business owner, display the hidden 'Share your story' button */
+function postButtonDisplay() {
   // Check user type
   var isBusinessOwner = localStorage.getItem("isBusinessOwner");
-  console.log("isBusinessOwner()", isBusinessOwner);
   if (isBusinessOwner == "true") {
-    console.log("is a business owner");
     displayElement("post-button");
   }
 }
```

Comment:
Review this function for this PR. Not sure when it was committed previously, this should be the diff comparison